### PR TITLE
Change `checkChangesMd` to be a string

### PR DESCRIPTION
**Pull request by omh1280**
Parse `repositories:{name}:checkChangesMd` as a string to be consistent across environment variables and JSON. __Iff it equals `"true"` will it be `true`.__

### CODE REVIEW

**Comment by omh1280** [comment]
@mramato Please review?

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The possible settings are:
 | `repositories:{name}:gitHubToken` | `string` | Token used to verify __outgoing__ requests to GitHub repository | ✓
 | `repositories:{name}:remindForum` | `boolean` | Enables the functionality to post a reminder message to a closed issue if it contains links to a Google Group forum. | X
 | `repositories:{name}:thirdPartyFolders` | `string` | Comma-seperated list of folders in which to look for changed files in pull request to remind user to update License. | X
-| `repositories:{name}:checkChangesMd` | `boolean` | If `true`, check if `CHANGES.md` has been updated in pull request. If not, post comment suggesting that it should be edited. | X
+| `repositories:{name}:checkChangesMd` | `string` | If `"true"`, check if `CHANGES.md` has been updated in pull request. If not, post comment suggesting that it should be edited. | X
```
Comment (mramato) [inline README.md]: I would still document this as a boolean.

File: index.js [hunk 2]
```diff
@@ -38,7 +38,10 @@ Settings.loadRepositoriesSettings('./config.json')
                 'User-Agent': 'cesium-concierge',
                 Authorization: 'token ' + repositorySettings.gitHubToken
             };
-            var checkChangesMd = defined(repositorySettings.checkChangesMd) ? repositorySettings.checkChangesMd : true;
+            var checkChangesMd = false;
+            if (defined(repositorySettings.checkChangesMd)) {
+                checkChangesMd = (repositorySettings.checkChangesMd === 'true');
```
Comment (mramato) [inline index.js]: I sent you an email with a link to a more complete parsing function you can use here.

**Comment by omh1280** [comment]
Thanks!

File: index.js [hunk 3]
```diff
@@ -38,9 +39,10 @@ Settings.loadRepositoriesSettings('./config.json')
                 'User-Agent': 'cesium-concierge',
                 Authorization: 'token ' + repositorySettings.gitHubToken
             };
-            var checkChangesMd = defined(repositorySettings.checkChangesMd) ? repositorySettings.checkChangesMd : true;
+            var checkChangesMd = parseBoolean(repositorySettings.checkChangesMd);
```
Comment (mramato) [inline index.js]: This should be handled in Settings, just like we do with thirdParty folders.  It should always be defined by this point.

File: lib/Settings.js [hunk 4]
```diff
@@ -43,6 +45,9 @@ Settings.loadRepositoriesSettings = function (configPath) {
         if (!defined(configJson[name]['gitHubToken'])) {
             return Promise.reject('repository ' + name + ' must have a `gitHubToken`');
         }
+        configJson[name]['checkChangesMd'] = parseBoolean(configJson[name]['checkChangesMd']);
```
Comment (mramato) [inline lib/Settings.js]: We want both of these settings to default to true, but the way parseBoolean works, these will default to false.

File: lib/Settings.js [hunk 5]
```diff
@@ -43,6 +45,9 @@ Settings.loadRepositoriesSettings = function (configPath) {
...
+        configJson[name]['remindForum'] = parseBoolean(configJson[name]['remindForum']);
```
Comment (mramato) [inline lib/Settings.js]: That and above should use `.` notation: `configJson[name].remindForum`

File: lib/Settings.js (hunk 4 above)
Comment (mramato) [inline lib/Settings.js]: You could fix this by doing something like 
```
configJson[name].checkChangesMd = parseBoolean(defaultValue(configJson[name].checkChangesMd, true));
```

**Comment by omh1280** [comment]
Bump!

**Comment by mramato** [comment]
Thanks.

**Pull request merged**